src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -365,7 +422,7 @@ private ParseUserDefinitionResult ParseUDFsAndNamedFormulas(string script, Parse
                 {
                     // = or ( expected here
                     ErrorTid(_curs.TokCur, TokKind.Equ);
-                    break;
+                    MoveToNextUserDefinition();

```

Comment:
>MoveToNextUserDefinition [](http://example.com/codeflow?start=20&length=24)

Why are we changing this?

Reply:
Ah, this was something that was missed in that optimization pass we did. This fails the whole parse on the first missing start of a function definition, but we should still move to the next udf and try that.